examples/telegram/interfaces/polling.py

```diff
@@ -0,0 +1,40 @@
+"""
+Polling
+=========
+
+The following example shows how to integrate your bot with the Pipeline API.
+
+Assuming that you already have a script that you need to deploy, all you need to do
+is to instantiante a TelegramMessenger and pass it to an `Interface` class.
+
+This class uses `PollingTelegramInterface` for local deployment with no webhooks.
+"""
+import os
+
+from dff.connectors.messenger.telegram.interface import PollingTelegramInterface, TelegramMessenger
+from dff.core.pipeline import Pipeline
+
+from dff.utils.testing.common import is_interactive_mode, run_interactive_mode, check_env_var
+from dff.utils.testing.toy_script import TOY_SCRIPT
+
+# Like Telebot, TelegramMessenger only requires a token to run.
+# However, all parameters from the Telebot class can be passed as keyword arguments.
+messenger = TelegramMessenger(os.getenv("BOT_TOKEN", "SOMETOKEN"))
+
+# For polling, you only need a `TelegramMessenger` instance.
+interface = PollingTelegramInterface(messenger=messenger)
+
+pipeline = Pipeline.from_script(
+    script=TOY_SCRIPT,  # Actor script object, defined in `.utils` module.
+    start_label=("greeting_flow", "start_node"),
+    fallback_label=("greeting_flow", "fallback_node"),
+    context_storage=dict(),
+    messenger_interface=interface,  # The interface can be passed as a pipeline argument.
```

Comment:
BTW: do you know that here and afterwards this particular line has no effect on documentation building and testing?
Example executed with `run_interactive_mode` function, that in place executes pipeline object with `pipeline.__call__()` method. While executing this method pipelines' `messenger_interface` property is __never used__. Is it OK with us?